libs/theme/src/lib/theme.ts

```diff
@@ -170,7 +170,8 @@ export const mixins = {
     font-family: 'DM Mono', monospace !important;
   `,
   mono: css`
-    font-family: 'Source Code Pro', monospace;
+    @import url('https://fonts.cdnfonts.com/css/vcr-osd-mono');
```

Comment:
Just tried & this doesn’t work for me :( Think we need to host the font elsewhere